pkg/query-service/utils/format.go

```diff
@@ -230,6 +230,22 @@ func ClickHouseFormattedValue(v interface{}) string {
 	}
 }
 
+func AddBackTickToFormatTag(str string) string {
+	if strings.Contains(str, ".") {
+		return "`" + str + "`"
```

Comment:
Is there any case where we end up adding backticks twice if the tag already was already formatter?